### PR TITLE
Add danacode type normalization

### PR DESCRIPTION
**Pull request by klngwll**
Add  danacode type normalization and spec.

### CODE REVIEW

File: whelk-core/src/main/resources/ext/marcframe.json [hunk 1]
```diff
@@ -5940,14 +5940,15 @@
       "include": ["identifier"],
       "match": [
         {
-          "when": "i1=7 & $2 =~ ansi|ANSI|doi|DOI|hdl|HDL|isan|ISAN|iso|ISO|istc|ISTC|iswc|ISWC|uri|URI|urn|URN",
+          "when": "i1=7 & $2 =~ ansi|ANSI|danacode|doi|DOI|hdl|HDL|isan|ISAN|iso|ISO|istc|ISTC|iswc|ISWC|uri|URI|urn|URN",
```
Comment (olovy) [inline whelk-core/src/main/resources/ext/marcframe.json:5943]: Check for uppercase `DANACODE` too?

Comment (klngwll) [reply whelk-core/src/main/resources/ext/marcframe.json:5943]: The code is lowercase and I checked _some_ examples, all where lowercase.

Comment (kwahlin) [reply whelk-core/src/main/resources/ext/marcframe.json:5943]: There is _one_ exception: https://libris.kb.se/3gc7x0vh1vnn5v99 :slightly_smiling_face:

**Review by olovy — APPROVED** [review]
:ship:

**Pull request merged**